build/azure-pipelines.yml

```diff
@@ -127,11 +127,36 @@ stages:
         minRiskLevel: 2
         onlyInputs: false
 
+- stage: Build_StubPackages
```

Comment:
You'll need to add something similar to .github/workflows/DevHome-CI.yml for PR validation.

Reply:
We decided not to add anything here, and will later add a similar PR validation in the DevHomeStub Repo